packages/tlsync/src/lib/TLSyncRoom.ts

```diff
@@ -234,7 +235,7 @@ export class TLSyncRoom<R extends UnknownRecord> {
 						lastChangedClock: 0,
 					},
 					{
-						state: PageRecordType.create({ name: 'Page 1', index: 'a1' }),
+						state: PageRecordType.create({ name: 'Page 1', index: 'a1' as IndexKey }),
```

Comment:
similar to David's comment `TLStore` - can this be `ZERO_INDEX_KEY`? if not, i'd love to understand the difference.